storage/innobase/log/log0recv.cc

```diff
@@ -2911,6 +2910,8 @@ recv_init_crash_recovery(void)
 	and restore them from the doublewrite buffer if
 	possible */
 
+	my_sd_extend_timeout(INNODB_EXTEND_TIMEOUT_INTERVAL);
```

Comment:
OK, I guess we need this to "tame down" systemd until `recv_recover_page_func()` makes its first report.